source-code/ide-extension/package.json

```diff
@@ -1,18 +1,18 @@
 {
-	"name": "@inlang/ide-extension",
+	"name": "inlang",
```

Comment:
the name is still inlang. it should be "vs-code-extension"

Reply:
That is weird:
![image](https://user-images.githubusercontent.com/1105080/225912372-c5feb18b-75ef-4765-b065-4ad9fa97a420.png)

https://github.com/inlang/inlang/pull/446/commits/742ad605e4f42b102e18f540af0d494437afe7a5

Reply:
Okay let's merge. Either it deplys or it doesn't